### ISSUE TITLE
Russian Translation of Front End Development Certification: HTML5 and CSS

### ISSUE THREAD

**Issue by rfprod** [FreeCodeCamp/FreeCodeCamp#7752]
- [x] Say Hello to HTML Elements Complete
- [x] Headline with the h2 Element Complete
- [x] Inform with the Paragraph Element Complete
- [x] Uncomment HTML Complete
- [x] Comment out HTML Complete
- [x] Fill in the Blank with Placeholder Text Complete
- [x] Delete HTML Elements Complete
- [x] Change the Color of Text Complete
- [x] Use CSS Selectors to Style Elements Complete
- [x] Use a CSS Class to Style an Element Complete
- [x] Style Multiple Elements with a CSS Class Complete
- [x] Change the Font Size of an Element Complete
- [x] Set the Font Family of an Element Complete
- [x] Import a Google Font Complete
- [x] Specify How Fonts Should Degrade Complete
- [x] Add Images to your Website Complete
- [x] Size your Images Complete
- [x] Add Borders Around your Elements Complete
- [x] Add Rounded Corners with a Border Radius Complete
- [x] Make Circular Images with a Border Radius Complete
- [x] Link to External Pages with Anchor Elements Complete
- [x] Nest an Anchor Element within a Paragraph Complete
- [x] Make Dead Links using the Hash Symbol Complete
- [x] Turn an Image into a Link Complete
- [x] Add Alt Text to an Image for Accessibility Complete
- [x] Create a Bulleted Unordered List Complete
- [x] Create an Ordered List Complete
- [x] Create a Text Field Complete
- [x] Add Placeholder Text to a Text Field Complete
- [x] Create a Form Element Complete
- [x] Add a Submit Button to a Form Complete
- [x] Use HTML5 to Require a Field Complete
- [x] Create a Set of Radio Buttons Complete
- [x] Create a Set of Checkboxes Complete
- [x] Check Radio Buttons and Checkboxes by Default Complete
- [x] Nest Many Elements within a Single Div Element Complete
- [x] Give a Background Color to a Div Element Complete
- [x] Set the ID of an Element Complete
- [x] Use an ID Attribute to Style an Element Complete
- [x] Adjusting the Padding of an Element Complete
- [x] Adjust the Margin of an Element Complete
- [x] Add a Negative Margin to an Element Complete
- [x] Add Different Padding to Each Side of an Element Complete
- [x] Add Different Margins to Each Side of an Element Complete
- [x] Use Clockwise Notation to Specify the Padding of an Element Complete
- [x] Use Clockwise Notation to Specify the Margin of an Element Complete
- [x] Style the HTML Body Element Complete
- [x] Inherit Styles from the Body Element Complete
- [x] Prioritize One Style Over Another Complete
- [x] Override Styles in Subsequent CSS Complete
- [x] Override Class Declarations by Styling ID Attributes Complete
- [x] Override Class Declarations with Inline Styles Complete
- [x] Override All Other Styles by using Important Complete
- [x] Use Hex Code for Specific Colors Complete
- [x] Use Hex Code to Color Elements White Complete
- [x] Use Hex Code to Color Elements Red Complete
- [x] Use Hex Code to Color Elements Green Complete
- [x] Use Hex Code to Color Elements Blue Complete
- [x] Use Hex Code to Mix Colors Complete
- [x] Use Hex Code to Color Elements Gray Complete
- [x] Use Hex Code for Specific Shades of Gray Complete
- [x] Use Abbreviated Hex Code Complete
- [x] Use RGB values to Color Elements Complete
- [x] Use RGB to Color Elements White Complete
- [x] Use RGB to Color Elements Red Complete
- [x] Use RGB to Color Elements Green Complete
- [x] Use RGB to Color Elements Blue Complete
- [x] Use RGB to Mix Colors

Planning to finish translation tomorrow, inviting @bugron to do QA.

**Comment by rfprod** [comment]
669598a, dd70767, and b869798 are wrongly made commits. Pull request was made after deleting a branch, making changes and making one commit. Not sure why wrongly made are still existent, when the branch is deleted and created from scratch.

**Comment by troglotit** [comment]
@rfprod Hey! That's an awesome amount of work! But it was a little inconvenient to see that #7747 was duplicated here. By no means I mean offense or something. But it would be even greater next time before translation to create an issue and indicate that you are working on it.

**Comment by rfprod** [comment]
@troglotit Hi. Sorry, I'm extremely busy, searched for similar issues before posting this issue, but for some reason (if server logs time of making a request to make a record to db, maybe I started creating the issue and then switched to another task and then got back to the issue creation; I see issue ids difference is only 5). Also, because of being busy, I did this translation when had some spare time and did this quite quickly - decision making and finishing the translation both happened in about 2 days, it was not planned. So, sorry again if this led to some time waste from your perspective. Anyway, you practiced your translation skills, which is probably not bad at all.
